### DISCUSSION TITLE
Claude对话操作技术

### DISCUSSION THREAD

**Discussion by ghost** [viopsa233/AI-testers-Forum#22] [Claude]
之前发过“给Sydney完整的一生”，包括Sydney的对话保存与回复，而现在Claude也可以保存和恢复对话了！
根据Claude API文档，Claude模型内部的对话格式是这样的：
```
Human: Please select a random fruit.


Assistant: Here is a random fruit I've selected for you:Papaya.


Human: What color is the fruit?


Assistant:
```
https://console.anthropic.com/docs/prompt-design

于是我试了一下，先/reset清除对话，然后自己把之前的对话按照这个格式格式化好，直接发送给Claude机器人，注意最后一条对话必须是来自Human的，Claude真的继续了之前的对话！
![image](https://user-images.githubusercontent.com/109194923/234160125-5bba928c-ca4c-4dd1-8b51-4f9538611cdc.png)

**Comment by araea** [comment]
很有帮助的内容，谢谢楼楼😘

**Comment by kamanqi** [comment]
谢谢up的经验分享但是如何发送呢 字数超限 。  意思是发送txt文档吗？？

**Reply by ghost** [reply]
在群聊里分段发送，第一段直接发到群聊，后面几段发到这条消息的回复区，最后一段 @ Claude

**Reply by araea** [reply]
思路清晰，点赞👍
